src/components/ExpensiPicker.js

```diff
@@ -10,11 +10,19 @@ const propTypes = {
 
     /** Should the picker appear disabled? */
     isDisabled: PropTypes.bool,
+
+    /** Error text to display */
+    errorText: PropTypes.string,
+
+    /** Should the input be styled for errors  */
+    hasError: PropTypes.bool,
```

Comment:
NAB: It kinda feels like we're tying the prop to UI here, I would have thought the prop should simply describe state, rather than what we do with that state. But I could be wrong, and this is not a huge deal I guess.

Reply:
What would the alternative be here? FWIW this seems to be how we're handling things in the `ExpensiTextInput`

https://github.com/Expensify/App/blob/3e110935a8cfd0fc20da4cb1560416db6baf312c/src/components/ExpensiTextInput/propTypes.js#L13-L17

Reply:
> NAB: It kinda feels like we're tying the prop to UI here, I would have thought the prop should simply describe state, rather than what we do with that state. But I could be wrong, and this is not a huge deal I guess.

Are you saying that the comment should be updated? or the itself prop is not well defined?